Userland/Utilities/ls.cpp

```diff
@@ -89,7 +89,7 @@ int main(int argc, char** argv)
         return 1;
     }
 
-    Vector<const char*> paths;
+    Vector<String> paths;
```

Comment:
Please squash this into the commit that it fixes.

Reply:
I squashed this fix into the original commit with the subject line `ls`. The `LibCore` commit remains separate though. Is that all right?

Reply:
That's fine, the LibCore change is its own change :)